.github/workflows/windows.yml

```diff
@@ -18,10 +18,26 @@ jobs:
         with:
           go-version: ${{ matrix.go-version }}
       - name: Checkout code
-        uses: actions/checkout@v2 
+        uses: actions/checkout@v2
       - name: Build Test
         run: |
           make smb-windows
       - name: Run Windows Unit Tests
         run: |
+          # start the CSI Proxy before running tests on windows
+          # TODO: (boddumanohar) CSI-Proxy team are not building the csi-proxy.exe. Currently I am using a binary that I built. In future we should change to offcial repo.
+          Start-Job -Name CSIProx -ScriptBlock {
+            Invoke-WebRequest https://github.com/boddumanohar/csi-proxy/releases/download/v0.2.1/csi-proxy.zip -OutFile csi-proxy.zip;
```

Comment:
could you use this binary: `https://kubernetesartifacts.azureedge.net/csi-proxy/v0.2.0/binaries/csi-proxy.tar.gz` ?